projects/requisition-management/src/app/pages/requisition-detail/requisition-summary/requisition-summary.component.html

```diff
@@ -28,6 +35,20 @@
           </ng-container>
         </ng-template>
       </dd>
+      <ng-container
+        *ngIf="
+          requisition.approval?.costCenterApproval?.approvers?.length && requisition.approval?.statusCode === 'PENDING'
+        "
+      >
+        <dt class="col-6 col-sm-4 col-md-3">
+          {{ 'approval.detailspage.cost_center.label' | translate }}
+          {{ 'approval.detailspage.approver.label' | translate: { '0': 1 } }}
```

Comment:
Is this combined label really a good idea? Are you certain this will work for all relevant languages?

Reply:
Tbh, I am not sure, but the alternative would be to create a lot of extra localization keys ( one for each time period ). Let's discuss it.